spec/routing/users_routing_spec.rb

```diff
@@ -1,31 +1,29 @@
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe UsersController, type: :routing do
-  describe "routing" do
-
-    it "routes to #index" do
-      expect(:get => "/users").to route_to("users#index")
+  describe 'routing' do
+    it 'routes to #index' do
+      expect(get: '/users').to route_to('users#index')
     end
 
-    it "routes to #show" do
-      expect(:get => "/users/1").to route_to("users#show", :id => "1")
+    it 'routes to #show' do
+      expect(get: '/users/1').to route_to('users#show', id: '1')
     end
 
-    it "routes to #create" do
-      expect(:post => "/users").to route_to("users#create")
+    it 'routes to #create' do
+      expect(post: '/users').to route_to('users#create')
     end
 
-    it "routes to #update via PUT" do
-      expect(:put => "/users/1").to route_to("users#update", :id => "1")
+    it 'routes to #update via PUT' do
+      expect(put: '/users/1').to route_to('users#update', id: '1')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.